### ISSUE TITLE
Still working?

### ISSUE THREAD

**Issue by Fdr3iZzz** [jlnbxn/Strync#1]
Hey, is this still maintained and working as I tryed to use it but it didnt work. It said that the server is runing while the localhost said that nothing is there. I did not put the apple stuff in the .inv so maybe thats why. Would be really awsome if we can get it to work as it seemed really promissing.
tyvm

**Comment by jlnbxn** [comment]
You are correct that the Apple token expired, the web version should be fixed again! Perhaps try revoking access of Strync in you Spotify [approved apps](https://www.spotify.com/us/account/apps/)  and then try logging in again if you're still experiencing issues. If that doesn't help, let me know.

**Comment by Fdr3iZzz** [comment]
Hey, thank you for fixing the web version, it works now. Is there an output for songs that couldnd be converted? I'm missing around 20 or so songs and don't want to check manually. I get a notification but it just says "undefined", not the title or something like that.

**Comment by jlnbxn** [comment]
Thanks for pointing that out, should be fixed! See: https://github.com/jlnbxn/Strync/commit/af54fba285c1cc7056d8de4a84f118ca9bb50da6

**Comment by Fdr3iZzz** [comment]
yup, its fixed. thank you very much. It works grate and looks really nice. Id suggest displaying the artist as well ^^. also, how do you host the website? im interestet :)
